07_Train_With_AML_Pipeline.ipynb

```diff
@@ -636,17 +641,141 @@
-    "rest_endpoint = published_pipeline.endpoint\n",
-    "print(\"You can perform HTTP POST on URL {} to trigger this pipeline\".format(rest_endpoint))\n",
-    "response = requests.post(rest_endpoint, \n",
-    "                         headers=aad_token, \n",
-    "                         json={\"ExperimentName\": experiment_name,\n",
-    "                               \"RunSource\": \"SDK\"})\n",
-    "run_id = response.json()[\"Id\"]\n",
-    "print(run_id)"
+    "# aad_token = auth.get_authentication_header()\n",
+    "# rest_endpoint = published_pipeline.endpoint\n",
+    "# print(\"You can perform HTTP POST on URL {} to trigger this pipeline\".format(rest_endpoint))\n",
+    "# response = requests.post(rest_endpoint, \n",
+    "#                          headers=aad_token, \n",
+    "#                          json={\"ExperimentName\": experiment_name,\n",
+    "#                                \"RunSource\": \"SDK\"})\n",
+    "# run_id = response.json()[\"Id\"]\n",
+    "# print(run_id)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Schedule the published pipeline to run regularly <a id='schedule_aml_pipeline'></a>\n",
+    "This step shows how to schedule the published pipeline to run regularly. This will also submit an initial run since a starting time for the schedule is not supplied."
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "schedule_name = \"hypetuning\"\n",
+    "frequency = \"Hour\"\n",
+    "interval = 1\n",
+    "recurrence = ScheduleRecurrence(frequency=frequency, interval=interval)\n",
+    "schedule = Schedule.create(\n",
+    "    workspace=ws,\n",
+    "    name=schedule_name,\n",
+    "    pipeline_id=published_pipeline.id,\n",
+    "    experiment_name=experiment_name,\n",
+    "    recurrence=recurrence,\n",
+    "    description=schedule_name)"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Write Out the URI <a id='write_uri_key'></a>\n",
+    "Write the URI to the statistics tracker."
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "statisticsCollector.addEntry(CollectionEntry.AKS_REALTIME_ENDPOINT,\n",
+    "                             published_pipeline.endpoint)"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "\n",
+    "Get a connection string to the workspace's storage to use to save the statistics."
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "storageConnString = \"YOUR_STORAGE_CONNECTION_STRING\""
```

Comment:
The published happy path deliberately does not create a scheduled pipeline. Doing so can leads to user problems and unexpected user costs. In addition, instrumentation should not be in the published path. These would be fine for the instrument branch.